app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/splash/SplashScreen.kt

```diff
@@ -0,0 +1,37 @@
+package com.kks.nimblesurveyjetpackcompose.ui.presentation.splash
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import com.kks.nimblesurveyjetpackcompose.R
+import com.ramcosta.composedestinations.annotation.Destination
+import com.ramcosta.composedestinations.navigation.DestinationsNavigator
+
+@Destination(start = true)
+@Composable
+fun SplashScreen(navigator: DestinationsNavigator) {
```

Comment:
Same comment re keeping PRs lean, this isn't needed yet 😄

Reply:
I need that annotation to navigate since I am using the library. 🙏

Reply:
@kaungkhantsoe Noted on the annotation, but the arg is still unused 😉